jest/jest-environment.js

```diff
@@ -0,0 +1,11 @@
+import JSDOMEnvironment from "jest-environment-jsdom";
+import cloneDeep from "lodash.clonedeep";
+
+// https://github.com/facebook/jest/blob/v29.4.3/website/versioned_docs/version-29.4/Configuration.md#testenvironment-string
+export default class FixJSDOMEnvironment extends JSDOMEnvironment {
+  constructor(...args) {
+    super(...args);
+    // https://github.com/jsdom/jsdom/issues/3363
+    this.global.structuredClone = cloneDeep;
```

Comment:
I am not sure how relevant/working this is, but here's a suggestion to use Node's structuredClone.
https://github.com/jsdom/jsdom/issues/3363#issuecomment-1921575184

Reply:
long story short - it broke a check that clojure uses under the hood: `structuredClone(array) instanceof Array` returned false